### PR TITLE
[WIP] Futurizes a ton of guns & ammo

### PR DESCRIPTION
**Pull request by Draganfrukts**
UPDATED FORM OF PR #12036 

## About The Pull Request

As said in the PR title - This aims to replace a bunch of absolutely outdated ammo and certain guns to be more in line with our setting. We've been trying to move away from the retrofuture thing forever and his is another step in the right way

## How This Contributes To The Skyrat Roleplay Experience

This makes the setting more believable while not interfering with gameplay in any major ways (and I'd doubt even minor ways). Except for some names and aesthetics, everything stays mostly the same but it has a sprinkle of 26th century on it.

## Changelog

All changes will be here when it's finished

:cl:
qol: Certain unspent ammo casings have bigger, full-sized sprites for ease of clicking on.
imageadd: Added unique casing sprites for 5.45
imageadd: Resprited AKM and Modern AKM
tweak: AKM and Modern AKM renamed to "Krinkov" and "Bubba's Krinkov". They're now chambered in 5.45x39mm 7N30 and are statistically the same weapons.
tweak: M16, Modern M16 and M16 Patriot renamed to "XM-2537", "XM-2537 SOPMOD" and "XM-2537 Amans Patriae" respectively.
spellcheck: 7.62, .50 BMG, .50 AE, 5.56, .45, 4.6x30, 7.62x39 M43 renamed to 5.8x42mm Mk. VII, .416 Stabilis, .454 Trucidator, .277 Aestus, .460 Ceres, 6.5x25mm and 5.45x39mm 7N30-B respectively.
spellcheck: AK projectile no longer incorrectly says 7.62x25mm
fix: XM2537 (M16) magazines no longer incorrectly hold 20 rounds instead of 30
code: Tidied up a couple of code comments in bulletrebelance and gunsgalore.
/:cl:

## Credits

CEV Eris - Bigger unspent cartridges (I used these as a base to modify:] )

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
immensely based pr

File: modular_skyrat/modules/bulletrebalance/code/sniper.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/projectile/bullet/p50
 	name =".50 BMG bullet"
```
Comment (Draganfrukts) [inline modular_skyrat/modules/bulletrebalance/code/sniper.dm]:
```suggestion
	name =".416 Stabilis bullet"
```
you're stupid gragonfurts

**Comment by Draganfrukts** [comment]
image
No more!

**Comment by Stalkeros2** [comment]
You remove 7.62x39 because it's old but then make AKs 5.45x39 which is practically equally old.
Curious.
jk 10 .277-chambered 21st century Black Mesa M16s/10 based PR

**Comment by Draganfrukts** [comment]
YOU HAVE TO BE FUCKING KIDDING ME

**Comment by Draganfrukts** [comment]
This was okayed to be put in as part of GUBMAN 3.4 instead of atomizing so we'll do that. :]

**Pull request closed without merging**